### PR TITLE
Fix disallowed html tags

### PR DESCRIPTION
**Pull request by Jing-song**

### CODE REVIEW

File: Commands/network/application-gateway/_list.md [hunk 1]
```diff
@@ -76,9 +76,8 @@ List application gateways.
         network application-gateway list -g MyResourceGroup
     ```
 
-### [2023-11-01](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5uZXR3b3JrL2FwcGxpY2F0aW9uZ2F0ZXdheXM=/2023-11-01.xml) **Stable**
+### [2023-11-01](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcmVzb3VyY2Vncm91cHMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5uZXR3b3JrL2FwcGxpY2F0aW9uZ2F0ZXdheXM=/2023-11-01.xml) **Stable**
 
-<!-- mgmt-plane /subscriptions/{}/providers/microsoft.network/applicationgateways 2023-11-01 -->
```
Comment (AllyW) [inline Commands/network/application-gateway/_list.md:81]: lost list by subscription here?

File: Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml [hunk 2]
```diff
@@ -359,8 +359,8 @@
...
           <help short="The name of the event hub. If none is specified, the default event hub will be selected."/>
         </arg>
-        <arg nullable="True" type="string" var="$parameters.properties.logAnalyticsDestinationType" options="log-analytics-destination-type" group="Properties">
-          <help short="A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type constructed as follows: <normalized service identity>_<normalized category name>. Possible values are: Dedicated and null (null is default.)"/>
+        <arg nullable="True" type="string" var="$parameters.properties.logAnalyticsDestinationType" options="log-analytics-destination-type log-ana-dtype" group="Properties">
+          <help short="A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type constructed as follows: `<normalized service identity>_<normalized category name>`. Possible values are: Dedicated and null (null is default.)"/>
         </arg>
```
Comment (AllyW) [inline Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml:364]: `log-ana-dtype` is typo?

File: Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml [hunk 3]
```diff
@@ -505,7 +505,7 @@
         </http>
       </operation>
       <operation>
-        <instanceUpdate instance="$Instance">
```
Comment (AllyW) [inline Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml:508]: this line should not be changed

File: Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml (hunk 2 above)
Comment (Jing-song) [reply Resources/mgmt-plane/L3tyZXNvdXJjZXVyaX0vcHJvdmlkZXJzL21pY3Jvc29mdC5pbnNpZ2h0cy9kaWFnbm9zdGljc2V0dGluZ3Mve30=/2021-05-01-preview.xml:364]: Fixed

**Pull request closed without merging**